src/current/v24.3/troubleshoot-lock-contention.md

```diff
@@ -0,0 +1,395 @@
+
+<img src="{{ 'images/v24.3/troubleshoot-lock-contention-high-contention.png' | relative_url }}" alt="High Contention" style="border:1px solid #eee;max-width:100%" />
+
+### Waiting statement
+
+To identify the exact statement in the transaction that experienced high contention, click the **Latest Transaction Execution ID**
+452da3e7-e7fa-4801-aa40-c59988d14eb6, the ID of the latest execution with the given [transaction fingerprint]({% link {{ page.version.version }}/ui-transactions-page.md %}).
+
+On the **Transaction Execution** page, navigate to the **Statement Executions** tab. In the list of statement executions, in the **Insights** column for `SELECT * FROM t where k = _`, there should be the **High Contention** insight. In Example 1, Transaction 2 had one statement (other than `SHOW database`). In a transaction with multiple statements, use this page to pinpoint the exact statement that experienced high contention.
+
+<img src="{{ 'images/v24.3/troubleshoot-lock-contention-waiting-statement.png' | relative_url }}" alt="Waiting statement" style="border:1px solid #eee;max-width:100%" />
+
+### Blocking transaction
+
+To identify the transaction that blocked **Transaction 2** and caused it to experience high contention, navigate back to the **Overview** tab.
+
+<img src="{{ 'images/v24.3/troubleshoot-lock-contention-overview-tab.png' | relative_url }}" alt="Overview tab" style="border:1px solid #eee;max-width:100%" />
+
+Scroll to the bottom of the Overview tab to the **Transaction with ID 452da3e7-e7fa-4801-aa40-c59988d14eb6 waited on** section which gives information about the blocking transaction.
+
+<img src="{{ 'images/v24.3/troubleshoot-lock-contention-blocking-transaction.png' | relative_url }}" alt="Blocking transaction" style="border:1px solid #eee;max-width:100%" />
+
+For more information about the blocking transaction, click the **Transaction Fingerprint ID** 08d8b7d97f830463 to open the [**Transaction Details** page]({% link {{ page.version.version }}/ui-transactions-page.md %}#transaction-details-page).
+
+<img src="{{ 'images/v24.3/troubleshoot-lock-contention-blocking-transaction-details.png' | relative_url }}" alt="Blocking transaction details" style="border:1px solid #eee;max-width:100%" />
+
+### Additional practice
+
+For Transaction 3, take steps similar to the above steps for Transaction 2, to identify the waiting statement that experienced high contention and the corresponding blocking transaction.
+
+## Remediate lock contention
+
+### Background context
+
+Locking conflicts are a natural artifact when business requirements call for concurrent data changes. Realistically, locking conflicts are unavoidable. The locking conflicts, however, are resolved efficiently with regard to the underlying resource utilization. When blocked transactions are waiting on a lock, they are not consuming CPU, disk, or network resources.
```

Comment:
I don't really understand what we're trying to say with the latter part of this paragraph

> The locking conflicts, however, are resolved efficiently with regard to the underlying resource utilization. When blocked transactions are waiting on a lock, they are not consuming CPU, disk, or network resources.

this may be technically true but seems irrelevant. the only reason i would be looking at this page is if i am coming here with a problem.  I mean if they're so efficient why do i have to come look at my database's locking docs instead of doing something else?

perhaps there is intent here that I'm missing, but as a reader this seems to say "blocked transactions are efficient, actually" and i'm like "if locking was fine and wasn't being a problem, i wouldn't be looking at these docs"

my suggestion is to drop the last two sentences and just go into the next paragraph, something like

> Locking conflicts are a natural artifact when business requirements call for concurrent data changes. Realistically, locking conflicts are unavoidable.

> Remediation is required when ...